vscode/src/common.ts

```diff
@@ -25,6 +25,16 @@ export function isQsharpDocument(document: TextDocument): boolean {
   );
 }
 
+// Returns true for all circuit documents
+export function isCircuitDocument(document: TextDocument): boolean {
+  return (
+    document.languageId === "qsc" &&
+    (Utils.extname(document.uri) === ".qsc" || document.isUntitled) &&
+    document.uri.scheme !== "git" &&
+    document.uri.scheme !== "pr"
+  );
+}
```

Comment:
I actually think you could go ahead and fold this into `isQsharpDocument`. i.e., just count `.qsc` files as Q# documents as well. I'm pretty sure all the callers of `isQsharpDocument` are also cases where we want to treat `.qsc` files as Q# files as well.

Reply:
done